### PR TITLE
edd_is_admin_page updates see #2657

### PR DESCRIPTION
**Pull request by chriscct7**
Implements `edd_is_admin` as defined in #2657

Supports new customer management ui and reports page

### CODE REVIEW

File: includes/admin/admin-pages.php [hunk 1]
```diff
@@ -45,37 +45,309 @@ function edd_add_options_link() {
...
+					$found = false;
+				}
+			} elseif ( in_array( $pagenow, $admin_pages ) ) {
+				$found = true;
+			}
+			break;
 	}
 
-	return (bool) apply_filters( 'edd_is_admin_page', $ret );
+	return (bool) apply_filters( 'edd_is_admin_page', $found, $page, $view );
```
Comment (pippinsplugins) [inline includes/admin/admin-pages.php]: `$passed_page` and `passed_view` need to be accessible here.

**Comment by pippinsplugins** [comment]
Something in here is failing.

Go to the main Downloads screen (post_type=download) and then run `var_dump( edd_is_admin_page() );`.

It will return `false`. It should be true.

**Comment by cklosowski** [comment]
@pippinsplugins looking

**Comment by pippinsplugins** [comment]
Nevermind, it's working. I forgot that Debug Bar Console fires through ajax so doesn't return expected results.

**Pull request merged**